stdlib/public/Concurrency/ExecutorAssertions.swift

```diff
@@ -238,19 +301,47 @@ extension GlobalActor {
 
 @available(SwiftStdlib 5.1, *)
 extension Actor {
-  /// A safe way to synchronously assume that the current execution context belongs to the passed in actor.
+  /// Assume that the current task is executing on this actor's serial executor,
+  /// or stop program execution otherwise.
+  ///
+  /// This method allows to *assume and verify* that the currently
+  /// executing synchronous function is actually executing on the serial
+  /// executor of this actor.
```

Comment:
```suggestion
  /// You call this method to *assume and verify* that the currently
  /// executing synchronous function is actually executing on the serial
  /// executor of this actor.
```

Trying to make the developer (or their code) the subject of the sentence, and avoid an awkward wording.  It might also be better as something like "to verify an assumption".  I'm not sure about the wording though, because the term "assume" doesn't frequently appear in APIs.